enterprise/server/invocation_search_service/invocation_search_service.go

```diff
@@ -170,6 +170,93 @@ func (s *InvocationSearchService) QueryInvocations(ctx context.Context, req *inp
 	if statusQuery != "" {
 		q.AddWhereClause(fmt.Sprintf("(%s)", statusQuery), statusArgs...)
 	}
+}
+
+func (s *InvocationSearchService) searchInvocationsInExecutions(ctx context.Context, req *inpb.SearchInvocationRequest) (*query_builder.Query, error) {
+	if s.oh == nil {
+		return nil, status.UnimplementedError("Execution filters require an OLAP DB.")
+	}
+	q := query_builder.NewQuery(`SELECT DISTINCT invocation_uuid FROM Executions as i`)
+	addCommonWhereClauses(q, req.GetQuery())
+	for _, f := range req.GetQuery().GetFilter() {
+		if f.GetMetric().Execution == nil {
+			continue
+		}
+		str, args, err := filter.GenerateFilterStringAndArgs(f, "i.")
```

Comment:
again - I think we need to make sure that "group_id" and "updated_at_usec" is added as part of the where clause, otherwise the performance is going to be bad.